src/Neptune.lean

```diff
@@ -1,35 +1,45 @@
 /-
 Lean bindings to the Neptune hash function
 -/
-import BinaryTools
+import Neptune.BinaryTools
 
 namespace Neptune
 
-constant NEPTUNE_OUT_LEN: Nat := 32
+/-
+BLS12-381 pairing-friendly elliptic curve construction
+-/
+namespace Blstrs
+constant SCALAR_BYTES_LE: Nat := 32
 
 /-
 A dependent ByteArray which guarantees the correct byte length.
 -/
-def NeptuneHash : Type := { r : ByteArray // r.size = NEPTUNE_OUT_LEN }
+def Scalar : Type := { r : ByteArray // r.size = SCALAR_BYTES_LE }
```

Comment:
Scalar is a field element? Should be mentioned in the doc comment if so

Reply:
Yes, but I don't understand it very well yet so I figured I would document it better once I know what it is used for.